helpdesk/overview/getting_started.rst

```diff
@@ -1,171 +1,67 @@
-   :align: center
+Team’s productivity and visibility
+----------------------------------
 
-From here, select "Configure SLA Policies" and click "Create".
+| Teams can have individual *Assignment Methods* to ensure tickets get redirected to the right
+  person:
 
-You will fill in information like the Helpdesk team, what the minimum
-priority is on the ticket (the stars) and the targets for the ticket.
+- *Manually*: tickets are manually assigned allowing employees to manage their own workload and
+  target tickets they are experts at;
+- *Random*: tickets are randomly assigned and everyone gets the same amount. This method ensures
+  that all tickets are handled as the assignment happens automatically;
+- *Balanced*: tickets are assigned to the person with the least amount of tickets so that everyone
+  fairly gets the same amount. Thereby, you ensure that all tickets get to be taken care of, as
+  the assignment is automatic.
 
-.. image:: media/helpdesk12.png
+.. image:: media/productivity_visibility.png
    :align: center
+   :alt: View of a helpdesk team settings page emphasizing the productivity and visibility features
+         in Odoo Helpdesk
 
-What if a ticket is blocked or is ready to be worked on?
---------------------------------------------------------
-
-If a ticket cannot be resolved or is blocked, you can adjust the "Kanban
-State" on the ticket. You have 3 options:
-
-- Grey - Normal State
+| For the *Random* and *Balanced* assignment methods, you are able to set the *Team Members* among
+  whom tickets are assigned. Leave the field empty to include all employees (with the proper
+  access rights).
+| The *Team Visibility* feature allows you to specify who can see and access the team’s tickets.
+  Therefore, ticket’s with sensible information, for example, are only seen by the right people.
+  Leave the field empty to include all employees (with the proper access rights).
 
-- Red - Blocked
+Set up stages and share it among teams
+======================================
 
-- Green - Ready for next stage
+Go to :menuselection:`Helpdesk --> Configuration --> Stages`. Create and/or edit stages as you need,
+and set specific teams to use certain stages under *Team*.
 
-Like the urgency stars you can adjust the state in the Kanban or on the
-Ticket form.
-
-.. image:: media/helpdesk13.png
+.. image:: media/stages_teams.png
    :align: center
+   :alt: View of a stage’s setting page emphasizing the option to add teams in Odoo Helpdesk
 
-How can my employees log time against a ticket?
------------------------------------------------
-
-First, head over to "Settings" and select the option for "Timesheet on
-Ticket". You will see a field appear where you can select the project
-the timesheets will log against.
+Stages can be shared between one or multiple teams, allowing you to adapt the pipeline to your
+individual needs. They also apply a visibility and access rule, as other teams are not able to see
+or use the stage.
```

Comment:
```suggestion
Stages can be shared between one or multiple teams, allowing you to adapt the pipeline to your
individual needs. They also apply visibility and access rules so other teams are not able to see
or use the stage.
```

Reply:
I'll keep it single. It is one action/rule, and keep the 'as' because it explains what is the visibility and access rule I am talking about.